pkg/visor/daemon.go

```diff
@@ -0,0 +1,45 @@
+package visor
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/SkycoinProject/skywire-mainnet/pkg/snet"
+
+	"github.com/SkycoinProject/dmsg/cipher"
+	cp "github.com/SkycoinProject/skycoin/src/cipher"
```

Comment:
I would leave it as `cipher`. `cp` is a bit confusing.

Reply:
Both github.com/SkycoinProject/dmsg/cipher and 	 "github.com/SkycoinProject/skycoin/src/cipher"
are imported in the same file.

How about skycoin_cipher "github.com/SkycoinProject/skycoin/src/cipher" ?

Reply:
@Kifen Oh, I haven't seen that. Sorry. Maybe `skycoincipher` and `dmsgcipher`? (Golang guidelines forbid the underscore)